index.js

```diff
@@ -5,7 +5,7 @@ module.exports = function (req, time) {
 		return req;
 	}
 
-	var delays = isNaN(time) ? time : {connect: time};
+	var delays = isNaN(time) ? time : {socket: time, connect: time};
```

Comment:
I'm still not convinced, that this is desired behaviour for `Number` in delays – many people will set timeout as Number and get false positive errors from servers, that accepts connection and take some time to process it.